nest.class.php

```diff
@@ -1464,11 +1464,11 @@ protected function doRequest($method, $url, $data_fields = NULL, $with_retry = T
             throw new RuntimeException("Error: Response from request to $url is not valid JSON data. Response: " . str_replace(array("\n","\r"), '', $response), NESTAPI_ERROR_NOT_JSON_RESPONSE);
         }
 
-        if ($info['http_code'] == 400) {
+        if ($info['http_code'] == 400 || $info['http_code'] == 500) {
```

Comment:
How about we throw an exception `if ($info['http_code'] >= 400)` instead? That should catch pretty much all possible cases.

Reply:
Sure. Update inbound